cpp/ps0007/p0007.soln.cpp

```diff
@@ -53,9 +56,13 @@ int main() {
             cout << "Invalid shape type." << endl;
             return -1;
     }
+
+    // check the validaty of the choice for the shape 1/1
+    // printing out the area of the shape 1/1
     double result = area(shape, dimensions);
     if (result != -1) {
         cout << "The area is: " << result << endl;
     }

```

Comment:
- [x] checked the user's input for choosing the shape
- [x] filled in the dimensions array with positive numbers 
- [x] invoked function to calculate the area of the shape
- [x] used suitable size of dimensions arrays with the shape
- [x] used `cout::std` in a right way to print out the area